index.css

```diff
@@ -3,8 +3,12 @@ html {
 }
 
 :root {
-  --square-size: 80px;
-  --captured-piece-size: 40px;
+  --square-size: 6vw;
+  --v-w: 11vw;
+  --full-size: 8vw;
+  --medium-size: 4rem;
+  --small-size: 3rem;
+  --captured-piece-size: 3vw;
```

Comment:
The inconsistent unit usage is a bit puzzling. I'd recommend sticking to `rem`/`em` by default. 

In regard to sizing the board, I think a better approach would be to define a single size variable and use the media queries to override the value, i.e.

```css
--square-size: 11vw;
--min-square-size: 3rem;
...
.board {
  width: calc(var(--square-size) * 8);
  ...
}
...
@media (min-width: 700px) {
  :root {
    --square-size: <idfk>
  }
```

this would allow you to declare the style (i.e `width: ...`) only once and resize the board by reassigning css variables

Reply:
I have things to say about this